optaplanner-examples/src/main/java/org/optaplanner/examples/nurserostering/persistence/NurseRosteringImporter.java

```diff
@@ -157,7 +157,7 @@ private void generateShiftDateList(NurseRoster nurseRoster,
             int maxDayIndex = Math.toIntExact(DAYS.between(startDate, endDate));
             int shiftDateSize = maxDayIndex + 1;
             List<ShiftDate> shiftDateList = new ArrayList<>(shiftDateSize);
-            shiftDateMap = new HashMap<>(shiftDateSize);
+            shiftDateMap = new LinkedHashMap<>(shiftDateSize);
```

Comment:
Is the `LinkedHashMap` here to keep stable order of elements?

Reply:
Yes. That is what all the other LinkedHashMaps do, too.